skey(3): shared library is in /lib; we don't install a profiling library

--- skey-1.1.5-orig/skey.3
+++ skey-1.1.5/skey.3
@@ -245,10 +245,8 @@
 .Bl -tag -width /usr/lib/libskey_p.a -compact
 .It Pa /usr/lib/libskey.a
 static skey library
-.It Pa /usr/lib/libskey.so
+.It Pa /lib/libskey.so
 dynamic skey library
-.It Pa /usr/lib/libskey_p.a
-static skey library compiled for profiling
 .El
 .Sh SEE ALSO
 .Xr skey 1 ,
